src/pull.go

```diff
@@ -22,6 +22,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"sort"
```

Comment:
Good point. A message is fine, but at least let's not return success by means of a non nil error. You can modify the function's return signature to (message string, err error) and then pass the message on.